README.md

```diff
@@ -153,6 +154,15 @@ Most files uploaded to the ENA FTP server need to be compressed.
 
 More information on how ENA wants to receive the files can be found [here](https://ena-docs.readthedocs.io/en/latest/submit/fileprep/preparation.html).
 
+**Note for data upload:** 
+Uploaded files persistently stored on the ENA server after upload for some time. 
+Thus, if multiple test submission are performed, it is possible to skip the data upload with `--no_upload` in
+subsequent submissions.
+This also allows uploading (large) datasets separately e.g. with [aspera](https://ena-docs.readthedocs.io/en/latest/submit/fileprep/upload.html).
+For the `--no_upload` argument,  data file(s) still with to be provided with `--data` 
```

Comment:
I think this sentence misses something :)

Reply:
Whoops, only had one coffee this morning ;)